api/apps/api/test/scenario-input-files/scenario-cost-surface/scenario-cost-surface.fixtures.ts

```diff
@@ -152,24 +152,24 @@ export const getFixtures = async () => {
     WhenGettingMarxanDataAsOwner: async () =>
       (
         await request(app.getHttpServer())
-          .get(`/api/v1/scenarios/${scenarioId}/marxan/dat/pu.dat`)
+          .get(`/api/v1/projects/${scenarioId}/marxan/dat/pu.dat`)
           .set('Authorization', `Bearer ${ownerToken}`)
       ).text,
     WhenGettingMarxanDataAsContributor: async () =>
       (
         await request(app.getHttpServer())
-          .get(`/api/v1/scenarios/${scenarioId}/marxan/dat/pu.dat`)
+          .get(`/api/v1/projects/${scenarioId}/marxan/dat/pu.dat`)
           .set('Authorization', `Bearer ${contributorToken}`)
       ).text,
     WhenGettingMarxanDataAsViewer: async () =>
       (
         await request(app.getHttpServer())
-          .get(`/api/v1/scenarios/${scenarioId}/marxan/dat/pu.dat`)
+          .get(`/api/v1/projects/${scenarioId}/marxan/dat/pu.dat`)
           .set('Authorization', `Bearer ${viewerToken}`)
       ).text,
     WhenGettingMarxanDataAsUserNotInScenario: async () =>
       await request(app.getHttpServer())
-        .get(`/api/v1/scenarios/${scenarioId}/marxan/dat/pu.dat`)
+        .get(`/api/v1/projects/${scenarioId}/marxan/dat/pu.dat`)
         .set('Authorization', `Bearer ${userNotInScenarioToken}`),
```

Comment:
this will need to stay as it was 

eason: https://github.com/Vizzuality/marxan-cloud/pull/1435/files#r1289957576, with apologies for the naming confusion that leads to think quite naturally that this is related to cost surfaces, while it isn't (_in practice_ the data that is served through this endpoint _is_ a cost surface, but that's just an accident)

Reply:
addressed in 09d815d9091d397ad86d4a5bb7d369abbe29f6e2